CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 2.18.0-rc.0 / 2020-04-24
+
+* [CHANGE] Federation: Only use local TSDB for federation (ignore remote read). #7077
+* [FEATURE] Tracing(!): Added experimental Jaeger support #7148
+* [ENHANCEMENT] Rules: `rule_evaluations_total` and `rule_evaluation_failures_total` have `rule_group` label now. #7094
```

Comment:
This is a CHANGE, as it'll break any users who were depending on this not having any instrumentation labels previously.

Reply:
cc @roidelapluie I had `CHANGE` previously (:

Reply:
ack